migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,84 @@
+[id="migration-from-version-3-to-4-overview"]
+= Migration from version 3 to 4 overview
+
+{product-title} 4 clusters are different from {product-title} 3 clusters. {product-title} 4 clusters contain new technologies and functionality that result in a cluster that is self-managing, flexible, and automated To know more about migrating from {product-title} 3 to 4 see, xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[migrating from version 3 to 4].
+
+[discrete]
+[id="mtc-3-to-4-overview-differences-mtc"]
+=== Differences between {product-title} 3 and 4
+Differences between {product-title} 3 and 4: To successfully transition to {product-title} 4, review the following information:
```

Comment:
Like in the other PR, I'd suggest removing the beginning snippets with the colon after it, like "Differences between {product-title} 3 and 4:" and just start with your normal sentence.

You could also consider adding an xref to the "differences" assembly, since you xref in most other section. You could also consider working in the "differences" wording into the sentence that you keep, since the one now talks about successfully transitioning, but this section is really about reviewing information prior to actually migrating.